pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -846,11 +847,13 @@ func TestDisabledUserAccount(t *testing.T) {
 	s := scheme.Scheme
 	err := apis.AddToScheme(s)
 	require.NoError(t, err)
+	config := configuration.LoadConfig()
+	require.NoError(t, err)
```

Comment:
this line is redundant - the error was already checked and `configuration.LoadConfig()` doesn't return an error
```suggestion
```

Reply:
Oops 😱, thanks for catching that